server/lib/nl_detection.py

```diff
@@ -0,0 +1,65 @@
+# The following classes are used for the NL Detection.
+from abc import ABC
+from dataclasses import dataclass
+from typing import Any, Dict, List
+
+
+@dataclass
+class Place:
+  """Place attributes."""
+  dcid: str
+  name: str
+
+
+@dataclass
+class SV:
+  """Statistical Variable attributes."""
+  dcid: str
+  name: str
+  place_type: str
+
+
+@dataclass
+class PlaceDetection:
+  """Various attributes of place detection."""
+  query: str
+  query_without_place: str
+  places_found: List[str]
+  main_place: Place
+
+
+@dataclass
+class SVDetection:
+  """Various attributes of SV detection."""
+  query: str
+  sv2scores: Dict[str, float]
+
+  highlight_svs: List[str]
+  relevant_svs: List[str]
+
+
+class ClassificationAttributes(ABC):
+  """Abctract class to hold classification attributes."""
+  pass
+
+
+@dataclass
+class SimpleClassificationAttributes(ClassificationAttributes):
+  """Simple classification attributes."""
+  sv_detected: SV
```

Comment:
should this have places?

Reply:
i have included it now. but the main place is in the PlaceDetection class any way and at this point we will only be using one main place. But it's ok to duplicate it here